drivers/auxiliary/wandererbox_pro_v3.cpp

```diff
@@ -0,0 +1,1328 @@
+    ENVMonitorNP[Probe2_Temp].setValue(temp2);
+    ENVMonitorNP[Probe3_Temp].setValue(temp3);
+    ENVMonitorNP[ENV_Humidity].setValue(DHTH);
+    ENVMonitorNP[ENV_Temp].setValue(DHTT);
+    ENVMonitorNP[DEW_Point].setValue((237.7 * ((17.27 * DHTT) / (237.7 + DHTT) + log((DHTH / 100)))) / (17.27 - ((17.27 * DHTT) / (237.7 + DHTT) + log((DHTH / 100)))));;
+    ENVMonitorNP.setState(IPS_OK);
+    ENVMonitorNP.apply();
+
+}
+
+
+void WandererBoxProV3::updatePower(double Tcurrent,double V19_current,double AR_current,double voltage)
+{
+    // Power Monitor
+    PowerMonitorNP[VOLTAGE].setValue(voltage);
+    PowerMonitorNP[TOTAL_CURRENT].setValue(Tcurrent);
+    PowerMonitorNP[V19_CURRENT].setValue(V19_current);
+    PowerMonitorNP[AR_CURRENT].setValue(AR_current);
+    PowerMonitorNP.setState(IPS_OK);
+    PowerMonitorNP.apply();
+}
+
+void WandererBoxProV3::updateUSB31(int res)
+{
+    usb31ControlSP[INDI_ENABLED].setState( (res== 1) ? ISS_ON : ISS_OFF);
+    usb31ControlSP[INDI_DISABLED].setState( (res== 0) ? ISS_ON : ISS_OFF);
+    usb31ControlSP.setState((res== 1) ? IPS_OK : IPS_IDLE);
+    usb31ControlSP.apply();
+}
+
+void WandererBoxProV3::updateUSB32(int res)
+{
+    usb32ControlSP[INDI_ENABLED].setState( (res== 1) ? ISS_ON : ISS_OFF);
+    usb32ControlSP[INDI_DISABLED].setState( (res== 0) ? ISS_ON : ISS_OFF);
+    usb32ControlSP.setState((res== 1) ? IPS_OK : IPS_IDLE);
+    usb32ControlSP.apply();
+}
+
+void WandererBoxProV3::updateUSB33(int res)
+{
+    usb33ControlSP[INDI_ENABLED].setState( (res== 1) ? ISS_ON : ISS_OFF);
+    usb33ControlSP[INDI_DISABLED].setState( (res== 0) ? ISS_ON : ISS_OFF);
+    usb33ControlSP.setState((res== 1) ? IPS_OK : IPS_IDLE);
+    usb33ControlSP.apply();
+}
+
+void WandererBoxProV3::updateUSB21(int res)
+{
+    usb21ControlSP[INDI_ENABLED].setState( (res== 1) ? ISS_ON : ISS_OFF);
+    usb21ControlSP[INDI_DISABLED].setState( (res== 0) ? ISS_ON : ISS_OFF);
+    usb21ControlSP.setState((res== 1) ? IPS_OK : IPS_IDLE);
+    usb21ControlSP.apply();
+}
+
+void WandererBoxProV3::updateUSB22(int res)
+{
+    usb22ControlSP[INDI_ENABLED].setState( (res== 1) ? ISS_ON : ISS_OFF);
+    usb22ControlSP[INDI_DISABLED].setState( (res== 0) ? ISS_ON : ISS_OFF);
+    usb22ControlSP.setState((res== 1) ? IPS_OK : IPS_IDLE);
+    usb22ControlSP.apply();
+}
+
+void WandererBoxProV3::updateDC34(int res)
+{
+    dc3_4ControlSP[INDI_ENABLED].setState( (res== 1) ? ISS_ON : ISS_OFF);
+    dc3_4ControlSP[INDI_DISABLED].setState( (res== 0) ? ISS_ON : ISS_OFF);
+    dc3_4ControlSP.setState((res== 1) ? IPS_OK : IPS_IDLE);
+    dc3_4ControlSP.apply();
+}
+
+void WandererBoxProV3::updateDC5(int res)
+{
+    dc5ControlNP[0].value = res;
+    dc5ControlNP.setState(IPS_OK);
+    dc5ControlNP.apply();
+}
+
+void WandererBoxProV3::updateDC6(int res)
+{
+    dc6ControlNP[0].value = res;
+    dc6ControlNP.setState(IPS_OK);
+    dc6ControlNP.apply();
+}
+
+void WandererBoxProV3::updateDC7(int res)
+{
+    dc7ControlNP[0].value = res;
```

Comment:
setValue(...). Just look for any .value in the code and replace that for setValue(..)